src/cue/k8s-aws.cue

```diff
@@ -0,0 +1,23 @@
+package katapult
+
+// AWS-specific definitions.
+
+k8s: service: [string]: {
+  annotations: {
+    "service.beta.kubernetes.io/aws-load-balancer-backend-protocol": string | *"http"
+    "service.beta.kubernetes.io/aws-load-balancer-ssl-ports": string | *"https"
+    "service.beta.kubernetes.io/aws-load-balancer-ssl-cert": "TODO" // arn:aws:acm:us-east-1:account:certificate/uuid
```

Comment:
And AWS-specific target adapter can be expressed as a separate set of definitions (potentially in a different cue package)